assets/js/modules/analytics-4/components/dashboard/ActivationBanner/SetupBanner.js

```diff
@@ -374,16 +374,17 @@ export default function SetupBanner( { onSubmitSuccess } ) {
 				</SpinnerButton>
 			}
 			footer={
-				<Fragment>
-					{ footerMessages.map( ( message, index ) => (
-						<p
-							className="googlesitekit-ga4-setup-banner__footer-text"
-							key={ index }
-						>
-							{ message }
-						</p>
-					) ) }
-				</Fragment>
+				footerMessages.length ? (
```

Comment:
I don't think we need the ternary here, something like:

```suggestion
				!! footerMessages.length && (
```

would work.